### DISCUSSION TITLE
How to restrict postgREST API for users on columns level and filtering level in postgres    ---Reg

### DISCUSSION THREAD

**Discussion by sureshbellamkonda96** [PostgREST/postgrest#3669] [Q&A]
Environment
PostgreSQL version: 15
PostgREST version:  v12.2.1 
Operating system: Windows

By default, users of the api served by postgrest can execute queries using all available operators and sort the results based on all available columns. I'd like to be able to restrict which operators can be used (and with which columns) as well as only enable read_only on specific columns as well entire columns data. 
 What's the recommended way of doing this?

**Comment by wolfgangwalther** [comment]
Looks like you already know about #2442, because big chunks of your text are copied from there...

**Reply by sureshbellamkonda96** [reply]
yeah, but i'm using windows platform can you please guide me on this.

**Reply by wolfgangwalther** [reply]
The referenced ticket is still open. Nothing in the "we don't currently support that" thing is specific to Windows or not.
